pulseapi/entries/views.py

```diff
@@ -248,7 +250,7 @@ def get_serializer_context(self):
     # When people POST to this route, we want to do some
     # custom validation involving CSRF and nonce validation,
     # so we intercept the POST handling a little.
-    @detail_route(methods=['post'])
+    @action(detail=True, methods=['post'])
```

Comment:
DRF 3.9 unified `detail_route` and `list_route` into a single `action` with a `detail` argument instead.